### PR TITLE
MOTECH-2574 documenting different DB names

### PR DESCRIPTION
**Pull request by craigappl**

### CODE REVIEW

**Comment by craigappl** [comment]
@llavoie Can you code review this?

File: docs/source/demos/different_DB_names.rst [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+The first step is to follow the :doc: `implementer's setup guide <../get_started/installing>`_ to create your default instance. You don't need to install any modules, but you'll need to make sure you complete the following steps:
+- When you complete the bootstrap form **make sure to choose Configuration Mode: File** 
+- After you click the verify button, you'll get an error about the web security activator not being able to start. That's because we need to modify some of the properties for file based configuration. Follow these steps to get everything configured correctly:
+
+.. code-block:: bash
+
+   #Stop the tomcat7 server
+   sudo service tomcat7 stop
+   #Create a file called motech-settings.properties that has the configuration variables
+   echo -e "system.language=en\nlogin.mode=repository" | sudo tee --append ~/.motech/config/motech-settings.properties
```
Comment (frankhuster) [inline docs/source/demos/different_DB_names.rst]: is sudo necessary here?

Comment (craigappl) [inline docs/source/demos/different_DB_names.rst]: I'm assuming that tomcat7 is installed as a service with permissions tomcat7:tomcat7 on the .motech folder. When I tried the command, it gave me permission denied without sudo and with 'sudo -u tomcat7', so I chose to add sudo then, 'chown' in the next line.

Comment (frankhuster) [inline docs/source/demos/different_DB_names.rst]: But who are you logged in as and where does ~ point to?
Wouldn't the .motech directory for tomcat installed as a service be somewhere else than ~/.motech?

**Comment by frankhuster** [comment]
LGTM

**Pull request merged**